server/huntflow_reloaded/handler.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, application, request, **kwargs):
         self._channel_name = self._channel_name or None
         self._decoded_body = {}
         self._handlers = {}
-        self._logger = logging.getLogger('tornado.application')
+        self._logger = Logger.with_default_handlers() #('tornado.application')
```

Comment:
* Remove the comment line.
* Pass `name='tornado.application'` to `with_default_handlers`.